api/UIEvent.json

```diff
@@ -50,18 +50,384 @@
           "deprecated": false
         }
       },
+      "UIEvent": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/UIEvent/UIEvent",
+          "description": "<code>UIEvent()</code> constructor",
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "11"
+            },
+            "firefox_android": {
+              "version_added": "14"
```

Comment:
I'm seeing "11" for both versions of Firefox.

Reply:
Firefox versions 1-3 and 11-13 were never released on mobile. See [here](https://github.com/mdn/browser-compat-data/pull/1715) for discussion.